### PR TITLE
Newhome

### PR DESCRIPTION
**Pull request by guenk**
Newhome with navbar and style

### CODE REVIEW

File: frontend/src/pages/Home/Style.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.titlesuggest {
+  display: flex;
+
+  padding-top: 10rem;
+  color: whitesmoke;
+  font-size: 2.5em;
+  padding-left: 50rem;
+
```
Comment (Sax593) [inline frontend/src/pages/Home/Style.scss]: Unnecessary space

File: frontend/src/pages/Home/Style.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+.logoxMS3 {
+  position: absolute;
+  height: 15em;
+  transform: rotate(-220deg);
+  right: -7%;
+}
+
+.titlesuggest {
+  display: flex;
+
```
Comment (Sax593) [inline frontend/src/pages/Home/Style.scss]: Unnecessary space

**Review by Sax593 — CHANGES_REQUESTED** [review]

**Review by Sax593 — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Comment by LoicBrassart** [comment]
Didn't see the conflicts, I'll have to re-read this later -_-

**Review by Sax593 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**